src/zc/recipe/cmmi/__init__.py

```diff
@@ -174,7 +176,7 @@ def build(self):
                             self.patch, is_temp = download(
                                 self.patch,
                                 md5sum=self.options.get('patch-md5sum'))
-                        except:
+                        except Exception:
```

Comment:
I don't think that's correct. We want to remove the temporary directory for all exceptions, including `BaseException` (like `KeyboardInterrupt`).

Reply:
Right.